ansible/playbooks/roles/compatibility_layer/defaults/main.yml

```diff
@@ -16,27 +16,25 @@ gentoo_prefix_path: /cvmfs/{{ cvmfs_repository }}/versions/{{ eessi_version }}/c
 gentoo_git_repo: https://github.com/gentoo/gentoo.git
 # Select a specific commit in the gentoo_git_repo that should be used for the bootstrap,
 # e.g. by checking: https://github.com/gentoo/gentoo/commits/master
-# April 17 (29492845e41ea6a0a4a9769c7e0ce287d106079b) commit is after fix for Lmod
-# gentoo_git_commit: 29492845e41ea6a0a4a9769c7e0ce287d106079b
-# June 8 (aab8473aa90e0287553b3348a5c5b17872df4b7b) commit that was current when fetching luaposix
-gentoo_git_commit: aab8473aa90e0287553b3348a5c5b17872df4b7b
+# March 7: latest commit (includes a RISC-V fix for GCC)
+gentoo_git_commit: 7e595dc3c3330522878452ea98906046c0d0f45b
 prefix_required_space: 15 GB
 prefix_user_defined_trusted_dirs:
   - "/cvmfs/{{ cvmfs_repository }}/host_injections/{{ eessi_version }}/compat/{{ eessi_host_os }}/{{ eessi_host_arch }}/lib"
 prefix_mask_packages: |
   # stick to GCC 10.x; using a too recent compiler in the compat layer complicates stuff in the software layer,
   # see for example https://github.com/EESSI/software-layer/issues/151
-  >=sys-devel/gcc-11
+  #>=sys-devel/gcc-11
   # mask OpenSSL 3.x, stick to OpenSSL 1.1.x for now to avoid problems with:
   # - older versions of Rust (see https://github.com/EESSI/software-layer/issues/257)
   # - older versions of cryptograhy in Python (see https://github.com/EESSI/software-layer/issues/258)
   >=dev-libs/openssl-3
 prefix_unmask_packages: |
-  # unmask older GCC to make it installable
-  =sys-devel/gcc-9*
+  # unmask older OpenSSL to make it installable
+  =dev-libs/openssl-1.1.1w
 prefix_bootstrap_use_flags: |
   # make sure that gold linker is installed with binutils
-  sys-devel/binutils gold
+  # sys-devel/binutils gold
```

Comment:
mention why (gold does not support RISC-V?)

Reply:
https://github.com/EESSI/compatibility-layer/pull/202/commits/4d5eb5b9476eb51c6c619fd189e749dd0107523a adds a comments.